ocs_ci/helpers/helpers.py

```diff
@@ -2961,3 +2962,43 @@ def get_mon_pdb():
     disruptions_allowed = pdb_obj.get().get("status").get("disruptionsAllowed")
     min_available_mon = pdb_obj.get().get("spec").get("minAvailable")
     return disruptions_allowed, min_available_mon
+
+
+def check_pvs_present_for_ocs_expansion(sc=constants.LOCALSTORAGE_SC):
+    """
+    Check for pvs present for OCS cluster expansion
+
+    Args:
+        sc (str): Name of SC
+
+    Raises:
+        Assertion if PVS not present
+
+    """
+    # TODO: need to handle it for N+1
+
+    for rack_no in range(0, 3):
+
+        pv_check_list = list()
+        nodes_obj = OCP(
+            kind=constants.NODE,
+            selector=f"{constants.TOPOLOGY_ROOK_LABEL}=rack{rack_no}",
+        )
+        nodes_data = nodes_obj.get()["items"]
+        node_name = [nodes["metadata"]["name"] for nodes in nodes_data]
+        logger.info(node_name)
+        for nodes in node_name:
+            pvs_obj = OCP(
+                kind=constants.PV, selector=f"{constants.HOSTNAME_LABEL}={nodes}"
+            )
+            pvs_data = pvs_obj.get()["items"]
+            for pv in pvs_data:
+                if pv["spec"]["storageClassName"] == sc:
+                    if pv["status"]["phase"] == constants.STATUS_AVAILABLE:
```

Comment:
In the pv file I mentioned above, https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/pv.py, there are helper functions to get the pv name and pv status